lib/ScriptFileStorage.js

```diff
@@ -58,19 +58,24 @@ $class.save = function(path, content, callback) {
  * @param {function(Object, string)} callback
  */
 $class.load = function(path, callback) {
-  fs.readFile(
-    path,
-    'utf-8',
-    function(err, content) {
-      if (err) return callback(err);
-
-      // remove shebang
-      content = content.replace(/^\#\!.*/, '');
-
-      var source = MODULE_HEADER + content + MODULE_TRAILER;
-      return callback(null, source);
-    }
-  );
+  var scriptId = this._scriptManager.findScriptIdByPath(path);
+  if (scriptId != null) {
+    this._scriptManager.getScriptSourceById(scriptId, callback);
```

Comment:
Pay attention that result from this._scriptManager.getScriptSourceById is an object not the content...

```javascript
this._scriptManager.getScriptSourceByIdFromCache(scriptId, function(error,data) {
        if (err) return callback(err);
        callback(null, data.scriptSource);
      });
```